.builders/scripts/utils.py

```diff
@@ -32,3 +39,112 @@ def extract_metadata(wheel: Path) -> email.Message:
             raise RuntimeError(message) from None
 
     return email.message_from_string(metadata_file_contents)
+
+
+
+def iter_wheels(source_dir: str) -> Iterator[Path]:
+    for entry in sorted(Path(source_dir).iterdir(), key=lambda entry: entry.name.casefold()):
+        if entry.suffix == '.whl' and entry.is_file():
+            yield entry
+
+def remove_test_files(wheel_path: Path) -> None:
+    '''
+    Unpack the wheel, remove excluded test files, then repack it to rebuild RECORD correctly.
+    '''
+    # First, check whether the wheel contains any files that should be excluded. If not, leave it untouched.
+    with ZipFile(wheel_path, 'r') as zf:
+        excluded_members = [name for name in zf.namelist() if is_excluded_from_wheel(name)]
+
+    if not excluded_members:
+        # Nothing to strip, so skip rewriting the wheel
+        return False
+
+    with tempfile.TemporaryDirectory() as td:
+        td_path = Path(td)
+
+        # Unpack the wheel into temp dir
+        unpack(wheel_path, dest=td_path)
+        unpacked_dir = next(td_path.iterdir())
+
+        # Remove excluded files/folders
+        for root, dirs, files in os.walk(td, topdown=False):
+            for d in list(dirs):
+                full_dir = Path(root) / d
+                rel = full_dir.relative_to(unpacked_dir).as_posix()
+                if is_excluded_from_wheel(rel):
+                    shutil.rmtree(full_dir)
+                    dirs.remove(d)
+            for f in files:
+                rel = Path(root).joinpath(f).relative_to(unpacked_dir).as_posix()
+                if is_excluded_from_wheel(rel):
+                    os.remove(Path(root) / f)
+
+        print(f'Tests removed from {wheel_path.name}')
+
+        # Repack to same directory, regenerating RECORD
+        pack(unpacked_dir, dest_dir=wheel_path.parent, build_number=None)
+
+    return True
+
+
+def is_excluded_from_wheel(path: str) -> bool:
+    '''
+    These files are excluded from the wheel in the agent build:
+    https://github.com/DataDog/datadog-agent/blob/main/omnibus/config/software/datadog-agent-integrations-py3.rb
+    In order to have more accurate results, this files are excluded when computing the size of the dependencies while
+    the wheels still include them.
+    '''
+    excluded_test_paths = [
+        os.path.normpath(path)
+        for path in [
```

Comment:
I'm wondering if we should define this in some toml file, so that it's easier for ppl to add more folders later.